s5_base/run.sh

```diff
@@ -86,7 +83,7 @@ if [ $stage -le 5 ]; then
   steps/train_mono.sh --boost-silence 1.25 --nj 5 --cmd "$train_cmd" \
     data/train_500short data/lang exp/mono
 
-  # TODO: Understand why we use lang_nosp here...
```

Comment:
Isn't this some wsj thing. I think we can just ignore it and call our lexicon dir lang

Reply:
It's been deleted. Don't worry about it.